.github/workflows/build.yml

```diff
@@ -63,7 +63,10 @@ jobs:
         name: artifacts
         path: ${{ env.ARTIFACTS_DIR }}
     - name: Build docker image
-      run: docker build -t ${{ secrets.DOCKER_DAPRIO_REGISTRY }}/${{ env.IMAGE_NAME }}:${{ env.REL_VERSION }} .
+      run: docker build -t ${{ env.IMAGE_NAME }}:${{ env.REL_VERSION }} .
```

Comment:
I had to separate the tag from build since it would fail for me in PR builds. Since tagging only matters for release, I think this is OK.